fbw-a32nx/src/localization/flypad/en.json

```diff
@@ -223,6 +223,10 @@
     },
     "Navigraph": {
       "AirportDoesNotExist": "Airport Does Not Exist",
+      "GoToThirdPartyOptions": {
+        "Title": "Go to 3rd Party Options to link Navigraph account",
+        "Button": "3rd Party Options"
+      },
```

Comment:
remember to add these strings to localezy

Reply:
Already done by @frankkopp I believe

Reply:
I did - I used the en.json from your branch - Localazy the adds all new keys in one go.